Atomic.md

```diff
@@ -0,0 +1,618 @@
+
+The sequence numbers will always be consumed (i.e. the `AccountRoot`'s `Sequence` will be incremented) for each processed inner transaction. A transaction counts as being "processed" if it is applied to the ledger, i.e. if a `tec` or `tes` error is received. The sequence number will be incremented by the total number of inner transactions that that account had included in the `Atomic` transaction, to avoid any hash collisions.
+
+## 5. Security
+
+### 5.1. Trust Assumptions
+
+Regardless of how many accounts' transactions are included in an `Atomic` transaction, all accounts need to sign the collection of transactions.
+
+#### 5.1.1. Single Account
+In the single account case, this is obvious; the single account must approve all of the transactions it is submitting. No other accounts are involved, so this is a pretty straightforward case.
+
+#### 5.1.2. Multi Account
+The multi-account case is a bit more complicated and is best illustrated with an example. Let's say Alice and Bob are conducting a trustless atomic swap via a multi-account `Atomic`, with Alice providing 1000 XRP and Bob providing 1000 USD. Bob is going to submit the `Atomic` transaction, so Alice must provide her part of the swap to him.
+
+If Alice provides a fully autofilled and signed transaction to Bob, Bob could submit Alice's transaction on the ledger without submitting his and receive the 1000 XRP without losing his 1000 USD. Therefore, the inner transactions must be unsigned. 
+
+If Alice just signs her part of the `Atomic` transaction, Bob could modify his transaction to only provide 1 USD instead, thereby getting his 1000 XRP at a much cheaper rate. Therefore, the entire `Atomic` transaction (and all its inner transactions) must be signed by all parties.
+
+## 6. Examples
+
+### 6.1. One Account
+
+In this example, the user is creating an offer while trading on a DEX UI, and the second transaction is a platform fee.
+
+#### 6.1.1. Sample Transaction
+
+<details open>
+<summary>
+
+The inner transactions are not signed, and the `AtomicSigners` field is not needed on the outer transaction, since there is only one account involved.
+</summary>
+
+```typescript
+{
+  TransactionType: "Atomic",
+  Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+  Flags: "1",
+  TxnIDs: [
+    "7EB435C800D7DC10EAB2ADFDE02EE5667C0A63AA467F26F90FD4CBCD6903E15E",
+    "EAE6B33078075A7BA958434691B896CCA4F532D618438DE6DDC7E3FB7A4A0AAB"
+  ],
+  RawTransactions: [
+    {
+      RawTransaction: {
+        TransactionType: "OfferCreate",
+        Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+        TakerGets: "6000000",
+        TakerPays: {
+          currency: "GKO",
+          issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
+          value: "2"
+        },
+        AtomicTxn: {
+          Account: "rUserBSM7T3b6nHX3Jjua62wgX9unH8s9b",
+          OuterSequence: 3,
+          BatchIndex: 0
+        },
+        Sequence: 0,
+        Fee: "0",
```

Comment:
This contradicts [above](#21-fee), which says
> each inner transaction handles its own fees.

(I won't point this out for the rest of the examples.)

Reply:
The linked line wasn't updated previously, so all the places were you marked this were actually the correct answer 😅